wallet/how-to/use-non-evm-networks/starknet/create-a-simple-starknet-dapp.md

```diff
@@ -132,27 +170,82 @@ This provides visual feedback to the user, confirming which wallet they are usin
 The following code is an example of how to update the interface with the connected wallet's details:
 
 ```typescript title="App.tsx"
```

Comment:
I suggest not adding the full app example here, just the snippet that displays wallet information. The full example is shown in the next step.

Reply:
okay. I removed the full example from here.